From aeee73cf67a56183f4dbae19e947ed81b1bb321d Mon Sep 17 00:00:00 2001
From: Alfred Chen <cchalpha@gmail.com>
Date: Sat, 7 May 2022 16:16:33 +0000
Subject: [PATCH 272/288] sched/alt: Dummy uclamp sync-up

88e4d8c3cf08 sched/uclamp: Fix iowait boost escaping uclamp restriction
d9f51b85bfd8 sched/sugov: Ignore 'busy' filter when rq is capped by uclamp_max
---
 kernel/sched/alt_sched.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/kernel/sched/alt_sched.h b/kernel/sched/alt_sched.h
index f2b9e686d6a6..7bbe006ce568 100644
--- a/kernel/sched/alt_sched.h
+++ b/kernel/sched/alt_sched.h
@@ -664,4 +664,14 @@ extern void sched_dynamic_update(int mode);
 #endif
 
 static inline void nohz_run_idle_balance(int cpu) { }
+
+static inline
+unsigned long uclamp_rq_util_with(struct rq *rq, unsigned long util,
+				  struct task_struct *p)
+{
+	return util;
+}
+
+static inline bool uclamp_rq_is_capped(struct rq *rq) { return false; }
+
 #endif /* ALT_SCHED_H */
-- 
2.37.0.rc0.15.g3b9a5a33c2

